dev_environment/l3af_on_windows.md

```diff
@@ -0,0 +1,85 @@
+### Prerequisites:
+
+- [Git](https://github.com/git-for-windows/git/releases/download/v2.41.0.windows.3/Git-2.41.0.3-64-bit.exe)
+- Cmake
+- Clang
+- nmake
+- [Golang](https://aka.ms/vs/17/release/vs_buildtools.exe)
+- C++ using: [MSYS2](https://www.msys2.org/)
+- [Visual Studio Build Tools 2022](https://aka.ms/vs/17/release/vs_buildtools.exe)
+
+### Steps to Build l3afd on Windows:
+
+1. Clone l3afd repo:
+```bash
+git clone https://github.com/l3af-project/l3afd.git
+```
+
+2. Build l3afd using:
+```bash
+cmake -B build
+cmake --build build
+```
+
+>l3afd.exe file will be created after this.
+
+### Steps to run l3afd on Windows:
+
+3. Update [l3afd_win.cfg](./cfg/l3afd_win.cfg) file:
+
+- Rename the file as `l3afd.cfg`
```

Comment:
I would mention that the l3afd.cfg in l3afd/config should be replaced with this config file

Reply:
Done